lib/upload/ecto.ex

```diff
@@ -96,7 +96,7 @@ if Code.ensure_loaded?(Ecto) do
       uploader = Keyword.get(opts, :with, Upload)
 
       changeset
-      |> put_change(field, key)
+      |> put_in([Access.key(:changes), field], key)
```

Comment:
I'm not sure how best to handle changes to forked external libraries, but I think they should be heavily documented. So you might want to add a big old comment here.